### PR TITLE
Getting Ganesha exports (bsc#1164497)

### PR DESCRIPTION
**Pull request by tbazant**
This update tells users how to view the configuration of Ganesha exports.

### CODE REVIEW

**Comment by tbazant** [comment]
lines 805-816 are just formatting outcome

File: xml/admin_nfsganesha.xml [hunk 1]
```diff
@@ -389,6 +405,80 @@ EXPORT {
     </variablelist>
    </sect3>
   </sect2>
+
+  <sect2 xml:id="ganesha-getting-exports">
+   <title>Getting Exports Configuration</title>
```
Comment (asettle) [inline xml/admin_nfsganesha.xml]: I get what this means, but it's not a great title. How about just "Exports Configuration" or "Exporting Configurations"

File: xml/admin_nfsganesha.xml [hunk 2]
```diff
@@ -389,6 +405,80 @@ EXPORT {
...
+   <para>
+    To get the exports configuration details, follow these steps:
```
Comment (asettle) [inline xml/admin_nfsganesha.xml]: What does it mean though "To get"? Are you downloading? Receiving? Creating? Exporting?

File: xml/admin_nfsganesha.xml [hunk 3]
```diff
@@ -389,6 +405,80 @@ EXPORT {
...
+      following command outputs a string of the
+      <replaceable>POOL_NAME</replaceable>/<replaceable>NAMESPACE</replaceable>
+      form.
+     </para>
+<screen>&prompt.cephuser;ceph dashboard get-ganesha-clusters-rados-pool-namespace
+cephfs_data/ganesha</screen>
+    </step>
+    <step>
+     <para>
+      Given the pool name and namespace, list the &rados; objects available on
```
Comment (asettle) [inline xml/admin_nfsganesha.xml]: "Given the pool name and namespace"... as in, you've received it from the above command? If so,

"Now you have the pool name and namespace, list the ..."

File: xml/admin_nfsganesha.xml [hunk 4]
```diff
@@ -389,6 +405,80 @@ EXPORT {
...
+      that pool:
+     </para>
+<screen>&prompt.cephuser;rados -p cephfs_data -N ganesha ls
+conf-osd-node1
+export-1
+conf-osd-node2</screen>
+     <tip>
+      <para>
+       To see how each node is configured, just view its content:
```
Comment (asettle) [inline xml/admin_nfsganesha.xml]:
```suggestion
       To see how each node is configured, view the content using the following command:

File: xml/admin_nfsganesha.xml [hunk 5]
```diff
@@ -389,6 +405,80 @@ EXPORT {
...
+       <literal>rados://cephfs_data/ganesha/export-1</literal>. If there are
+       multiple configurations, each node can use a different configuration.
+      </para>
+     </tip>
+    </step>
+    <step>
+     <para>
+      Each export configuration is stored in a single object with the name
+      <literal>export-<replaceable>ID</replaceable></literal>. Use the
+      following command to get the contents of the object and save it to
```
Comment (asettle) [inline xml/admin_nfsganesha.xml]: Same comment about "to get" applies here.

File: xml/admin_nfsganesha.xml (hunk 1 above)
Comment (tbazant) [reply xml/admin_nfsganesha.xml]: 'Exports Configuration' was the previous section and it dealt with how to CREATE a configuration for a NFS export.
This one is bout how to GET the configuration for already existing exports as it is not transparent in a config file but rather  RADOS object.

File: xml/admin_nfsganesha.xml (hunk 2 above)
Comment (tbazant) [reply xml/admin_nfsganesha.xml]: my previous reply clears this

File: xml/admin_nfsganesha.xml (hunk 3 above)
Comment (tbazant) [inline xml/admin_nfsganesha.xml]:
```suggestion
      By using the received pool name and namespace, list the &rados; objects available on
```

File: xml/admin_nfsganesha.xml (hunk 5 above)
Comment (tbazant) [reply xml/admin_nfsganesha.xml]: i know, maybe 'obtain' is generally better here?

File: xml/admin_nfsganesha.xml (hunk 1 above)
Comment (asettle) [reply xml/admin_nfsganesha.xml]: Then I think "get" is too vague, as it didn't convey your meaning to me at all. @lproven is usually good at the wordy words here, but at a stab, how about: "Retrieving the Exports Configuration" or "Recover Exports Configuration"

File: xml/admin_nfsganesha.xml (hunk 5 above)
Comment (asettle) [reply xml/admin_nfsganesha.xml]: Yep, I like that!

Comment (lproven) [reply xml/admin_nfsganesha.xml]: Agreed. "Get" is a very vague word which has dozens of meanings. How about "retrieve" or "obtain"?

**Pull request merged**